roles/forklift_versions/molecule/default/verify.yml

```diff
@@ -20,8 +20,8 @@
     - name: Ensure versions have been set correctly
       assert:
         that:
-          - foreman_repositories_version == 'nightly'
-          - katello_repositories_version == 'nightly'
+          - foreman_repositories_version == '3.17'
```

Comment:
2.2 ;)

Reply:
Oh yea, I just wanted you to see your testing get put through it's paces ;)